### PR TITLE
add scikit-learn and scikit-optimize

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related: https://github.com/wolfi-dev/os/pull/4733

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0
- [ ] Patch source: _patch source here_

### CODE REVIEW

File: py3-scikit-learn.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package:
+  name: py3-scikit-learn
+  version: 1.3.0
+  epoch: 0
+  description: A set of python modules for machine learning and data mining
+  copyright:
+    - license: new BSD
```
Comment (rawlingsj) [inline py3-scikit-learn.yaml:7]:
```suggestion
    - license: BSD-3-Clause
```

File: py3-pillow.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+package:
+  name: py3-pillow
+  version: 10.0.0
+  epoch: 0
+  description: Python Imaging Library (Fork)
+  copyright:
+    - license: custom:PIL
```
Comment (rawlingsj) [inline py3-pillow.yaml:7]: We don't have any `- license: custom:PIL`  packages, @kaniini I couldn't find this on https://spdx.org/licenses/, is this the correct format?

Comment (kaniini) [reply py3-pillow.yaml:7]: It appears to be https://spdx.org/licenses/HPND.html

**Review by imjasonh — DISMISSED** [review]

**Review by kaniini — CHANGES_REQUESTED** [review]
Please fix the `custom:` license tags as previously discussed.

File: py3-cycler.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package:
+  name: py3-cycler
+  version: 0.11.0
+  epoch: 0
+  description: Composable style cycles
+  copyright:
+    - license: BSD
```
Comment (kaniini) [inline py3-cycler.yaml:7]:
```suggestion
    - license: BSD-3-Clause
```

File: py3-scikit-optimize.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
+package:
+  name: py3-scikit-optimize
+  version: 0.9.0
+  epoch: 0
+  description: Sequential model-based optimization toolbox.
+  copyright:
+    - license: BSD 3-clause
```
Comment (kaniini) [inline py3-scikit-optimize.yaml:7]:
```suggestion
    - license: BSD-3-clause
```

Comment (developer-guy) [reply py3-scikit-optimize.yaml]: good catch, thanks.

**Review by kaniini — APPROVED** [review]

**Pull request merged**